src/install/python/python-options/python-agent-uvicorn.mdx

```diff
@@ -0,0 +1,19 @@
+---
+componentType: appInfo
+optionType: framework
+headingText: Review start-up steps using Uvicorn
+---
+
+You need to integrate the Python agent with your application so that your app's major functions and web requests are received by the agent. To integrate the Python agent with your app, you can run the newrelic-admin script in front of your usual app startup command, or you can use the Python agent API.
+
+If you start your app with `python app.py` and use our Python agent **version 5.20.0.149** or higher, you can use the admin script integration method. The admin script was included when you installed the Python agent.
+
+Here's an example:
+
+```
+NEW_RELIC_CONFIG_FILE=<var>path/to/</var>newrelic.ini newrelic-admin run-program uvicorn path_to_app
+```
+
+<Callout variant="tip">
+  If you have difficulty with this automatic initialzation, you can also try our manual instrumentation steps. To get those instructions, go back to the first question and select **Unsupported WSGI framework**.
```

Comment:
Uvicorn is an ASGI server not a WSGI server. This should be pointing to Unsupported ASGI framework.